engine/src/main/java/org/terasology/persistence/serializers/GsonSerializer.java

```diff
@@ -36,14 +36,31 @@
 import java.io.StringWriter;
 import java.io.Writer;
 
-// TODO: Document
+/**
+ * GsonSerializer provides the ability to serialize and deserialize objects to and from JSON
+ * <br>
+ * Serialized JSONs can be forwarded/written to various output types <br>
```

Comment:
I think JSON is uncountable (like water) so shouldn't be pluralized, but I wouldn't die on this hill.